components/layout.js

```diff
@@ -122,7 +122,7 @@ Layout.propTypes = {
 };
 
 Layout.defaultProps = {
-  backgroundColor: "#e3e3e3"
+  backgroundColor: "#eaebed"
```

Comment:
We've added a selection of colours to `theme.js`, you can use`theme.colour.paleGrey` here